### PR TITLE
Implement class relationships tests

### PR DESCRIPTION
**Pull request by svilenvelikov**
Implemented Test initial state scenario

### CODE REVIEW

File: test-cypress/integration/explore/class.relationships.spec.js [hunk 1]
```diff
@@ -0,0 +1,97 @@
+describe('Class relations screen validation', () => {
+
+    let repositoryId;
+
+    before(() => {
+        repositoryId = 'class-relations-repo' + Date.now();
+        cy.createRepository({id: repositoryId});
+        cy.visit('/repository');
```
Comment (mihailradkov) [inline test-cypress/integration/explore/class.relationships.spec.js]: Not needed anymore

File: test-cypress/integration/explore/class.relationships.spec.js [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+        getRelationsToolbar().should('be.visible').within(() => {
+            cy.get('.remove-all-classes-btn').should('be.visible');
+            cy.get('.reload-diagram-btn').should('be.visible');
+            cy.get('.export-diagram-btn').should('be.visible');
+        });
+        // Class relations list should be visible and have 10 classes selected
+        getDependenciesList().should('be.visible').within(() => {
+            cy.get('.item .row.active').should(($rows) => {
+                expect($rows).to.have.length(10);
+            })
```
Comment (mihailradkov) [inline test-cypress/integration/explore/class.relationships.spec.js]: ; :stuck_out_tongue:

File: test-cypress/integration/explore/class.relationships.spec.js [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+        verifySelectedDirectionFilter('all');
+        // The toolbar and its buttons should be visible
+        getRelationsToolbar().should('be.visible').within(() => {
+            cy.get('.remove-all-classes-btn').should('be.visible');
+            cy.get('.reload-diagram-btn').should('be.visible');
+            cy.get('.export-diagram-btn').should('be.visible');
+        });
+        // Class relations list should be visible and have 10 classes selected
+        getDependenciesList().should('be.visible').within(() => {
+            cy.get('.item .row.active').should(($rows) => {
```
Comment (mihailradkov) [inline test-cypress/integration/explore/class.relationships.spec.js]: or `should('have.length', 10')` ?

Comment (svilenvelikov) [reply test-cypress/integration/explore/class.relationships.spec.js]: I've search for something like this with no luck. Thanks.

**Pull request merged**